### PR TITLE
Fixed last of the warnings in the Sphinx Build

### PR DESCRIPTION
**Pull request by intelkevinputnam**
Fixes include the following:
1. Fixed :option: usage in concepts/restart.rst
1. Fixed broken link in /concepts/swupd-about.rst
1. Fixed mis-labaled code-blocks in maintenance/bulk-provision.rst
1. Fixed :option: usage in guides/maintenance/mixer.rst
1. Fixed broken link in guides/maintenance/swupd-guide.rst
1. Fixed broken links in reference/collaboration/documentation/documentation.rst
1. Updated :option: usage guidelines in reference/collaboration/documentation/inline.rst
1. Added page that was not included in TOC in reference/reference.rst
1. Fixed :option: usage in tutorials/azure.rst
1. Fixed :option: usage in tutorials/docker/docker.rst
1. Fixed mis-labeled code-blocks in tutorials/fmv.rst and replaced hard coded line numbers with auto generated
1. Fixed footnotes in tutorials/multi-boot/multi-boot.rst
1. Fixed :option: usage in tutorials/telemetry-backend/telemetry-backend.rst
1. Added suppress directive to conf.py (it is currently commented out)

### CODE REVIEW

**Comment by MCamp859** [comment]
I only see 1 warning when I make html, is this real or user error?
C:\Users\Mary C\Documents\zzmaryfork\clear-linux-documentation\source\clear-linux\guides\maintenance\autospec.rst:70: WARNING: Problem with "start-after" option of "include" directive:
Text not found.

File: source/clear-linux/reference/collaboration/documentation/inline.rst [hunk 1]
```diff
@@ -34,20 +34,29 @@ the roles.
```
Comment (mvincerx) [inline source/clear-linux/reference/collaboration/documentation/inline.rst]: Hello @intelkevinputnam.
Starting at Line 21, it appears the legacy version of this doc had several alignment errors. While you're at it, would you mind realigning indents Line 21 and thereafter to fit cascading logic?

**Comment by intelkevinputnam** [comment]
@mvincerx @MCamp859 Should be fixed now. Please take a look. Yes, the autospec.rst warning should be fixed in the merge.

**Comment by MCamp859** [comment]
LGTM

**Comment by mvincerx** [comment]
@intelkevinputnam.  When I view the rendered copy of your last commit, I still see minor indentation errors. Easy to fix. Let's discuss in our meeting.

**Comment by intelkevinputnam** [comment]
I'll figure out a way to solve the indent warnings and add a commit.

**Comment by mvincerx** [comment]
LGTM. Great collaboration on this one, everyone. @intelkevinputnam - Thanks for your perseverance on this complex PR.  Reminder: All: Please test the `make html` to test if more warning messages on build.

**Pull request merged**